packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2472,18 +2478,19 @@ export default class Meeting extends StatelessWebexPlugin {
         this.meetingFiniteStateMachine.fail(error);
         LoggerProxy.logger.error('Meeting:index#join --> Failed', error);
 
-        joinFailed(error);
-        this.deferJoin = undefined;
-
+        // TODO:  change this to error codes and pre defined dictionary
         Metrics.sendOperationalMetric(
           METRICS_OPERATIONAL_MEASURES.JOIN_FAILURE,
           {
             correlation_id: this.correlationId,
-            locus_id: this.locusUrl.split('/').pop(),
```

Comment:
Is the Locus ID no longer necessary for this metric?

Reply:
its not necessary once you join a meeting